.circleci/test_docker_container.sh

```diff
@@ -2,6 +2,12 @@
 
 set -xeuo pipefail
 
+# test for working en_US.UTF-8 locale
```

Comment:
Can we add some echo messages here so that is is clear in the logs what is actually going on here?  Mostly just a line saying validate utf8 locale configured correctly.